SteamRPC.js

```diff
@@ -5,45 +5,42 @@ import SteamID from "steamid";
 import fetch from "node-fetch";
 import LogUpdate from "log-update";
 
+import config from "./config.js";
 
-// ==================== Configuration Settings =====================
-// Steam user identification
-
-// You'll need to find your own SteamID64 URL using https://steamrep.com
-// Note: This also allows custom URLs like https://steamcommunity.com/id/crementif but they require providing a valid web key.
-const steamProfileURL = "https://steamcommunity.com/profiles/76561198259089872";
-
-// ONLY needs to be replaced if you use a custom URL in the steamProfileURL variable above. There's no real benefit!
-// You can get one from https://steamcommunity.com/dev/apikey. Use localhost as the domain name.
-const steamWebKey = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
-
-// Advanced Configuration Settings
-const pollRate = 20*1000; // Poll Steam Rich Presence every n seconds. You should keep this at or above 20 seconds.
-const discordClientId = "433731243516100629"; // Keep this id unless you want to supply your own application id.
 
 // =================================================================
 // Initialize Status Variables
 let gameStatus = "";
-let discordStatus = "logging in";
+let discordStatus = "waiting for supported game...";
 let steamStatus = "obtaining user id";
 let debugLine = "";
+let version = "1.0";
 
 // Initialize Discord Objects
-const discordRPCClient = new DiscordRPC.Client({
-    clientId: discordClientId,
-    transport: "ipc"
-});
-
-// Setup discord RPC
-discordRPCClient.on("ready", () => {
-    discordStatus = "connected (IPC)";
-    renderWindow();
-});
+var discordRPCClient = null;
 
-discordRPCClient.on("error", (err, message) => {
-    discordStatus = `error ${err}: ${message}`;
-    renderWindow();
-});
+//todo let presences keep track of timestamp
+var lastTimestamp = null;
+var lastStatus;
```

Comment:
I would use let, even if they're globals, just because it makes it fit in more.
```suggestion
let discordRPCClient = null;

//todo let presences keep track of timestamp
let lastTimestamp = null;
let lastStatus;
```